packages/react/package.json

```diff
@@ -73,6 +73,8 @@
     "write-pkg": "^4.0.0"
   },
   "dependencies": {
+    "@heroicons/react": "1.0.6",
```

Comment:
I tried several ways to get the Hero V1 icons in as they are very old and do not work like the modern ones we are using currently.  This can go away also.